recipes/libtensorflow/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set version = "1.13.1" %}
+
+package:
+  name: libtensorflow-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/tensorflow/tensorflow/archive/v{{ version }}.tar.gz  # [linux]
+  sha256: 7cd19978e6bc7edc2c847bce19f95515a742b34ea5e28e4389dade35348f58ed  # [linux]
+
+  patches:
+    - patches/0001-add-rt-to-linkopts.patch  # [linux]
+    - patches/0002-stable-image-rotate-test.patch  # [linux]
+    - patches/0003-relax-tolerence-in-special_math_test.patch  # [linux]
+    - patches/0004-skip-read-only-debugger_cli_common-test.patch  # [linux]
+    - patches/0005-fix-lookup_ops-test.patch  # [linux]
+    - patches/0006-relax-precision-in-matrix_log_op_test.patch  # [linux]
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [linux]
+    - {{ compiler('cxx') }}  # [linux]
+    - bazel >=0.19.0,<=0.21.0  # [linux]
+  host:
+    # conda build requirements
+    - swig  # [linux]
+    - zlib  # [linux]
+
+outputs:
+  - name: libtensorflow
+    script: cp_libtensorflow.sh  # [unix]
+    test:
+      commands:
+        - test -f $PREFIX/lib/libtensorflow.so  # [not win]
+
+  - name: libtensorflow_cc
+    script: cp_libtensorflow_cc.sh  # [unix]
+    requirements:
+      host:
+        - libprotobuf
+        - eigen
+        - nsync
+        - abseil-cpp
```

Comment:
Are these host requirements needed?  Other than eigen they do not seem used in the install script.

Reply:
I have removed all host requirements.